draft-ietf-ppm-dap.md

```diff
@@ -2155,14 +2138,50 @@ structure includes the following:
   reports included in the batch, such that the interval's start and duration are
   both multiples of the task's `time_precision` parameter. Note that in the case
   of a time-interval query (see {{batch-mode}}), this interval can be smaller
-  than the one in the corresponding `CollectionReq.query`.
+  than the one in the corresponding `CollectionJobReq.query`.
 
 * `leader_encrypted_agg_share`: The Leader's aggregate share, encrypted to the
   Collector.
 
 * `helper_encrypted_agg_share`: The Helper's aggregate share, encrypted to the
   Collector.
 
+If the Leader finds the `CollectionJobReq` to be valid, it immediately responds
+with HTTP status 201 Created with a body containing a `CollectionJobResp`. The
+`status` field is set to `processing`, and the response contains a Location
+header containing the relative reference
+`/tasks/{task-id}/collection_jobs/{collection-job-id}`. The Leader SHOULD
+include a Retry-After header field to suggest a polling interval to the
+Collector.
+
+After receiving the response to its `CollectionJobReq`, the Collector makes an
+HTTP GET request to the aforementioned Location to check on the status of the
```

Comment:
```suggestion
After receiving the response to its `CollectionJobReq`, the Collector periodically makes
HTTP GET requests to the aforementioned Location to check on the status of the
```

I think we should highlight that this is a repeated polling operation -- as currently written, the start of this sentence reads as if only a single HTTP GET is sent.

Reply:
Also, the response might have had status "finished", in which case there's no need to poll?

Reply:
> Also, the response might have had status "finished", in which case there's no need to poll?

I added a blurb to the paragraph `Once both aggregate shares are successfully obtained`. Let me know if this reads clearer.